resources/helm/v2.1/global.yaml

```diff
@@ -1,444 +0,0 @@
-# Global and common settings for installing Istio.
```

Comment:
We may want to consider keeping this file, as it serves as global default values for all charts.  This will help maintain consistency.  That said, if upstream are manually updating individual values.yaml files for each chart, we run the risk of missing a change.  catch 22

Reply:
Upstream already moved all the values from `global.yaml` to `values.yaml` and I think we already got those changes in rebase. I have also updated scripts to change values in `values.yaml` instead of `global.yaml`.

Upstream PR: https://github.com/istio/istio/issues/25184